spec/features/user_logs_out_spec.rb

```diff
@@ -20,14 +20,14 @@
 
     click_button 'Log in'
 
-    expect(page).to have_content('Signed in successfully')
+    expect(page).to have_content('Logged in successfully')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.